pontoon/terminology/models.py

```diff
@@ -34,14 +61,25 @@ class Term(models.Model):
         "auth.User", models.CASCADE, related_name="terms", null=True, blank=True
     )
 
+    objects = TermQuerySet.as_manager()
+
+    def translation(self, locale):
+        if self.do_not_translate:
+            return self.text
+        else:
+            try:
+                return self.translations.filter(locale=locale).first().text
```

Comment:
Can there be several translations for a single term? If so, what makes you sure that the `first` is the one to use? If not, shouldn't it be better to use something like `get` or another function that ensures there's a single result?

Reply:
There can only be one translation per locale. I use [`first()`](https://docs.djangoproject.com/en/3.0/ref/models/querysets/#django.db.models.query.QuerySet.first), because it acts like `get_or_None()`, which I prefer over `get_or_raise`.

Would you rather see I use `get()` and catch `TermTranslation.DoesNotExist`?

Reply:
I'm not strongly opinionated there. Maybe @jotes has arguments for one way or the other?

Reply:
I think that `.get()` is more suitable here because it informs about the expected object/DB constraints. I'm not sure if that's consistent with the current conventions of error-handling in the codebase, but that would be my personal pick.